### ISSUE TITLE
Strange glitch in timeline

### ISSUE THREAD

**Issue by sofokli1** [toggl/superday#641]
As seen in the gif below, when scrolling to the top and then scrolling down a bit and not removing the finger from the screen some strange movements happen.
![jun-21-2017 17-27-27](https://user-images.githubusercontent.com/885122/27392537-380d022c-56a7-11e7-90e4-d429ebfbd618.gif)

**Comment by sofokli1** [comment]
Here i just grab the work slot that starts at 5:36 PM and scroll up and down a few times, i did not relese my finger and in a few seconds the glitch happens
![jun-21-2017 17-39-34](https://user-images.githubusercontent.com/885122/27393143-be9c1ade-56a8-11e7-8667-31da272b2056.gif)

**Comment by Odrakir** [comment]
Yeah, I've seen this before... it only happens in today's timeline, right?

**Comment by sofokli1** [comment]
I think so, can't reproduce in other day

**Comment by sofokli1** [comment]
I think this was fixed + can't reproduce

**Comment by sofokli1** [comment]
i am seeing something similar again

**Comment by amulware** [comment]
(Let's not delay the current release because of this. A fix would still be very welcome.)

**Comment by sofokli1** [comment]
This shows how the issue appear after expanding a slot. The `scroll to top` behaviour in the end of the gif happens without touches.

![aug-09-2017 15-07-40](https://user-images.githubusercontent.com/885122/29120780-8df78b74-7d14-11e7-8679-4a9e328a3f40.gif)

**Comment by Odrakir** [comment]
After all changes to the timeline, is this still a thing, @sofokli1? Can it be closed?

**Comment by sofokli1** [comment]
I still see it. I think it is related to [this line here](https://github.com/toggl/superday/blob/develop/teferi/UI/Modules/Timeline/TimelineViewModel.swift#L56)
